src/pages/Pay.tsx

```diff
@@ -139,17 +139,19 @@ const Pay = () => {
     );
 
     return (
-        <div data-status={status()} class="frame">
-            <SettingsCog />
-            <h2>
-                {t("pay_invoice", { id: params.id })}
-                <Show when={swap()}>
-                    <SwapIcons swap={swap()} />
-                </Show>
-            </h2>
+        <div data-status={status()} class="frame" id="pay">
```

Comment:
```suggestion
        <div data-status={status()} class="frame">
```
Not being used as far as I can tell

Reply:
@michael1011 leftover from a previous attempt, removed!